### DISCUSSION TITLE
Versioning or Roadmap?

### DISCUSSION THREAD

**Discussion by dbready** [jgm/djot#20] [General]
First of all - love this. A principled, light-weight markup unsaddled by baggage, and designed by someone with the chops to appreciate the nuances of every decision.

I was wondering how stable you considered the current implementation. How far is this from a 1.0 release? What would it take for this to become fully integrated into Pandoc? Do you hope for this to eventually replace Pandoc Markdown?

**Comment by jgm** [comment]
The current version is definitely "alpha."  I want to get more feedback and think about things some more before saying it's stable in any way.

If djot starts getting used by people, we could integrate it into pandoc.  It would take Haskell implementations of the parser and writer (or we could cheat and just load the Lua). Full integration can wait, though, because we already have bidirectional conversion with pandoc, via djot-reader.lua and djot-writer.lua.

In any case pandoc would continue to support pandoc markdown.

**Comment by tbdalgaard** [comment]
I really enjoy DJOt. So I hope others will take it and use it. I already prefer Djot over Pandoc's Markdown, when reading and writing, so I hope this project will continue. If I can't contribute with code, how can a more regular user help?

**Reply by jgm** [reply]
You can help by using it, letting others know about it, and reporting any rough edges.

**Comment by dereuromark** [comment]
Maybe we don't start with `1.0`, but 0.1, 0.2, etc. Others like TOML did the same.
This way we can see the progress, can clearly define the differences and communicate also for tooling what they are compatible with.

I just saw that the first 2 already exist: https://github.com/jgm/djot/releases

@jgm Would it work to make a milestone with 0.3 tickets to "close/finish"? And then set a rough estimate/timeframe.
This way people can at least get a feeling of it slowly becoming usable.

The main issue right now is that there is no official contract other than the current specs, which could be "moving".
